### PR TITLE
Changes the button order in the note's menu

### PR DESCRIPTION
**Pull request by lzimann**
Now it is "Add Note - Add message - Add to watchlist".

### CODE REVIEW

**Comment by Jordie0608** [comment]
Hello Sir/Madam
Please provide reasoning for the PR per our contributing rules:
> Please explain why you are submitting the pull request, and how you think your change will be beneficial to the game. Failure to do so will be grounds for rejecting the PR.

Thanks, the tgstation development team

**Comment by bgobandit** [comment]
why: because of muscle memory, admins leave notes way more than they leave messages but message is the first option

**Comment by lzimann** [comment]
what @bgobandit said

**Comment by Jordie0608** [comment]
I didn't expect an actual response, let alone a serious one

File: code/modules/admin/sql_message_system.dm [hunk 1]
```diff
@@ -255,9 +255,9 @@
 					notedata += data
 		output += "<h2><center>[target_ckey]</center></h2><center>"
 		if(!linkless)
+			output += " <a href='?_src_=holder;addnote=[target_ckey]'>\[Add note\]</a>"
 			output += "<a href='?_src_=holder;addmessage=[target_ckey]'>\[Add message\]</a>"
```
Comment (Jordie0608) [inline code/modules/admin/sql_message_system.dm]: Change the spacing here, the space in the now first line should be moved to this one.

Comment (lzimann) [inline code/modules/admin/sql_message_system.dm]: oops

**Pull request merged**